### DISCUSSION TITLE
Ajouter les traitements conseillés ?

### DISCUSSION THREAD

**Discussion by Jeoffreybauvin** [iopool/community#19] [Ideas]
Bonsoir,

Ce serait pas mal que l'API renvoie également la liste des traitements conseillés plutot que juste un booléen hasAnActionRequired :).

Merci,

**Comment by mguyard** [comment]
Bonjour,

Oui j'avais fait cette demande entre autre il y a 3 ans à la sortie de l'API :
https://github.com/iopool/community/discussions/4
